apps/server/src/index.ts

```diff
@@ -95,7 +95,14 @@ async function bootstrapServer() {
   const chartsService = new ChartsServiceImpl(chartsRepository)
 
   // Exchanges
-  const exchangeService = new MockExchangeServiceImpl()
+  const exchangeRepository = new CoinMarketCapRepository(ds)
+  const exchangeService = new ExchangeServiceImpl(exchangeRepository, ds)
+
+  exchangeService.fetchExchangeRates()
+  // TBD
+  setInterval(() => {
+    exchangeService.fetchExchangeRates()
+  }, 240000)
```

Comment:
?

We don't need to put setInterval, just be lazy, i.e: we grab the tickers if redis returns empty (as we had configured ttl to max 5 mins), everytime that value is empty, when a client request that data, we query it.

No need to keep polling with a setTimeout